src/styles/styles.js

```diff
@@ -2874,6 +2874,37 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    joinRoomPromptContainer: {
+        maxHeight: 92,
+        height: 92,
+        borderRadius: 18,
```

Comment:
The border radius from Figma is 20. Bonus if we can use a variable for this.

Reply:
Turns out there already is! `variables.componentBorderRadiusRounded`, I've updated it to use this.